Tests/MongoSwiftSyncTests/MongoDatabaseTests.swift

```diff
@@ -191,6 +191,40 @@ final class MongoDatabaseTests: MongoSwiftTestCase {
         expect(listNamesEvents[3].command["nameOnly"]).to(equal(false))
     }
 
+    func testListCollectionsComment() throws {
+        let client = try MongoClient.makeTestClient()
+        let monitor = client.addCommandMonitor()
+        let db = client.db(Self.testDatabase)
+
+        // comment only supported here for 4.4+
+        guard try client.serverVersionIsInRange("4.4", nil) else {
+            print("Skipping list collections comment test due to unsupported server version")
+            return
+        }
+
+        // clear out collections
+        try db.drop()
+        let comment = BSON("commenter")
+
+        _ = try db.createCollection("foo")
+        _ = try db.createCollection("bar")
+        _ = try db.createCollection("baz")
+
+        try monitor.captureEvents {
+            let options = ListCollectionsOptions(comment: comment)
+            _ = Array(try db.listCollections(options: options))
```

Comment:
since were not actually looking at the values, we can omit the `Array(...)`

Reply:
omitted cast